src/modules/esl-carousel/plugin/nav/esl-carousel.nav.dots.less

```diff
@@ -0,0 +1,28 @@
+// TODO: refactor
+// DOTS navigation
+.esl-carousel-dots {
+  display: block;
+  text-align: center;
+  opacity: 1;
+  transition: opacity 0.5s ease-out;
+  cursor: default;
+
+  .carousel-dot {
+    display: inline-block;
+    width: 9px;
+    height: 9px;
```

Comment:
Will it be useful to define the size of dots using CSS variables?

Reply:
Yep! Thanks! 
Good point for dots reimplementation ticket.